### ISSUE TITLE
[Feature Request] Repository Manager Favourites Folder

### ISSUE THREAD

**Issue by fourofspades** [ForkIssues/Tracker#1484]
There are currently 2 nodes, Repositories and Recent, it would be nice if we had a 3rd Favourites, or Pinned, or whatever you want to call it, where specific repos can be added.

**Comment by DanPristupov** [comment]
Hi,

You can create a folder, call it `.favourites` and put any repos there.

**Comment by fourofspades** [comment]
Yes, I am aware of this, but it looks naff, as it's not a top level node, but a folder under all repositories.  It looks inconsistent with everything else.  (I have no other folders, this would be the only one, and it's yellow compared to everything else)

Shouldn't Pinned or Favourites be a 1st class citizen?
